src/Entity/DataPolicy.php

```diff
@@ -16,41 +17,95 @@
  * @ContentEntityType(
  *   id = "data_policy",
  *   label = @Translation("Data policy"),
- *   label_collection = @Translation("Data policy consents"),
  *   handlers = {
- *     "list_builder" = "Drupal\gdpr_consent\DataPolicyListBuilder",
+ *     "storage" = "Drupal\gdpr_consent\DataPolicyStorage",
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "views_data" = "Drupal\gdpr_consent\Entity\DataPolicyViewsData",
+ *     "translation" = "Drupal\gdpr_consent\DataPolicyTranslationHandler",
```

Comment:
Perhaps use `Drupal\content_translation\ContentTranslationHandler` as the `DataPolicyTranslationHandler` is empty?

Reply:
@bramtenhove, I will check it.

Reply:
I've been testing translations. After removing this I got errors when enabling content translation for Data Policy.
Maybe put this back?